src/components/Icon/index.js

```diff
@@ -0,0 +1 @@
+export {default} from "./Icon.js";
```

Comment:
```suggestion
export { default } from './Icon.js';
```
lint 규칙에 맞게 정리하면 이렇습니다.

Reply:
감사합니다.ㅠ.ㅠ